p2p/rlpx.go

```diff
@@ -507,8 +507,8 @@ func readHandshakeMsg(msg plainDecoder, plainSize int, prv *ecdsa.PrivateKey, r
 	return buf, s.Decode(msg)
 }
 
-// importPublicKey unmarshals 512 bit public keys.
-func importPublicKey(pubKey []byte) (*ecies.PublicKey, error) {
+// ImportPublicKey unmarshals 512 bit public keys.
+func ImportPublicKey(pubKey []byte) (*ecies.PublicKey, error) {
```

Comment:
Do you still need to rename this function so it's "exportable"?  When I do a search for this function in this PR diff, I don't see it being used.